config/default.yml

```diff
@@ -231,6 +231,7 @@ RSpec/FilePath:
   IgnoreMethods: false
   SpecSuffixOnly: false
   VersionAdded: '1.2'
+  VersionUpdated: '1.40'
```

Comment:
It's `VersionChanged` if I remember correctly (or if it was changed to VersionUpdated, the whole file should be updated as well)

Reply:
Nice catch, thanks!
Fixed.